packages/slopShell/PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=slopShell
+pkgver=1
+_SLOP_DIR="/opt/$pkgname-$pkgver/"
+pkgrel=1
+pkgdesc='PHP based backdoor/webshell with C2 capabilities'
+arch=('any')
+groups=('blackarch', 'blackarch-backdoor')
+url='https://github.com/oldkingcone/slopShell'
+license=('Apache')
+depends=('postgresql-server', 'php', 'nginx', 'postgresql', 'php-pear', 'php-fpm')
+makedepends=('git')
+source=("git+https://github.com/oldkingcone/$pkgname.git")
+# without .git directory
+sha512sums=('4A0F0EAADF1EEE376148847CC4C5B49FCFD494192C2F059F3BA5ECB499E2C41B')
+
+prepare() {
+if ! [ -e $_SLOP_DIR ];then
+    cd $pkgname
+    echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+    sudo git clone $url $_SLOP_DIR && sudo chown -R $( whoami ):postgres $_SLOP_DIR
+else
+    echo "Directory is already created."
+fi
+$( which pg_ctl ) initdb -D $_SLOP_DIR/slop_data -l $_SLOP_DIR/main.log
+cd $_SLOP_DIR/slop_data && $( which createdb ) sloppy_bots -E utf-8 -O $( whoami ) -h localhost -p 5432 -U postgres
+sudo $( which chmod ) 770 /var/run/postgresql
+sudo $( which chown ) postgres:postgres /var/run/postgresql
+$( which pg_ctl ) start -D $_SLOP_DIR/slop_data -l $_SLOP_DIR/main.log
+echo "If any of these commands failed, you will need to re-run them yourself."
+echo "Creating a setup script in your home directory."
+curl https://raw.githubusercontent.com/oldkingcone/slopShell/master/setup.sh -o ~/slop_setup.sh
```

Comment:
The user shouldn't run a script afterwards, to install the actual tool, but the PKGBUILD should take care of these steps